routes/persona.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const connectDB = require('../db');
+connectDB();
+const Persona = require('../models/persona');
+const personaRouter = express.Router(); 
+
+//listar personas
+personaRouter.get(``, async (_req, res) => {
```

Comment:
se ve mas lindo asi? se ve raro poner `` sin variables dentro, lo mismo para las otras

```suggestion
personaRouter.get('/', async (_req, res) => {
```